spec/features/search_peeps_spec.rb

```diff
@@ -0,0 +1,20 @@
+
+feature 'search by keyword' do
+  scenario 'it filters by keyword' do
+    add_peep('breaking news', 'I made eggs today')
```

Comment:
really clean how you've used your methods instead of SQL directly in your feature tests.

Reply:
Agreed!